server/src/main/java/com/google/android/attestation/ParsedAttestationRecord.java

```diff
@@ -96,10 +96,23 @@ private ParsedAttestationRecord(
     this.teeEnforced = teeEnforced;
   }
 
-  public static ParsedAttestationRecord createParsedAttestationRecord(X509Certificate cert)
+  public static ParsedAttestationRecord createParsedAttestationRecord(X509Certificate[] certs)
```

Comment:
shall we use X509Certificate... so that it works with List or array ?

Reply:
An array is fine (we need to incorporate this example into the library anyway, we can do it then).